ega-data-api-res/src/main/java/eu/elixir/ega/ebi/reencryptionmvc/service/internal/LocalEGAArchiveServiceImpl.java

```diff
@@ -38,10 +40,8 @@
 
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
-import java.util.AbstractMap;
-import java.util.Arrays;
-import java.util.Base64;
-import java.util.Map;
+import java.math.BigInteger;
+import java.util.*;
```

Comment:
I'd prefer to not use star imports if possible

Reply:
Done